package.json

```diff
@@ -102,15 +104,12 @@
     "three": "0.87.0",
     "typeahead.js": "^0.11.1",
     "typedoc": "^0.11.1",
-    "typescript": "^2.0.0",
```

Comment:
@filippomc typescript is used by flexlayout, not needed?

Reply:
We shouldn't need to specify library dependencies in our package.json. There was a specific reason to have that?

Reply:
Anyway I did not remove that on purpose, I suppose there was something wrong with the merges.
Another related point: had we needed that, it should have been a devDependency, not a dependency.

Reply:
We need to ask @ddelpiano when he's back, I always assume that dependencies that are added are needed but certainly it should be a dev one.